Separation of repository configuration and installation

beechesII opened the pull request:
Sometimes you want to configure your repositories without using this role. With `vector_manage_repo` is it possible to deactivate repo creation.

beechesII committed:
separation of repo configuration and installation

beechesII committed:
fix ansible-lint action

beechesII committed:
fix var-naming[no-role-prefix]: Variables names from within roles should use vector_ as a prefix. (register: download_repo_key)

beechesII committed:
bump version to 2.0.0

schurzi reviewed (changes requested):
Why are the tests not running?

Also I think https://github.com/telekom-mms/ansible-vector/blob/32be5175d85b917a63c93b8f7ec658e6623ae117/molecule/vector/converge.yml#L16 needs the new variable also set to `true`.

beechesII committed:
add vector_manage_repo to molecule test case

beechesII commented:
> Why are the tests not running?
> 
> Also I think
> 
> https://github.com/telekom-mms/ansible-vector/blob/32be5175d85b917a63c93b8f7ec658e6623ae117/molecule/vector/converge.yml#L16
> 
> needs the new variable also set to `true`.

Molecule Tests were disabled by GitHub because we changed nothing since 60 Days. 

thx for the hint. I added `vector_manage_repo` to converge.yml.
